go.mod

```diff
@@ -36,7 +36,7 @@ require (
 	github.com/iancoleman/strcase v0.2.0
 	github.com/illarion/gonotify v1.0.1
 	github.com/insomniacslk/dhcp v0.0.0-20221215072855-de60144f33f8
-	github.com/josharian/native v1.0.1-0.20221213033349-c1e37c09b531
```

Comment:
We still have usages? Or indirect?

Reply:
Indirect. So I want to make sure that if any of our indirect ones use it, at least they use the version I bumped to use x/sys/cpu